plugin-places/src/main/java/com/mapbox/mapboxsdk/plugins/places/picker/ui/PlacePickerActivity.java

```diff
@@ -143,7 +143,7 @@ public void onCameraMoveStarted(int reason) {
   @Override
   public void onCameraIdle() {
     Timber.v("Map camera is now idling.");
-    markerImage.animate().yBy(Math.abs(markerImage.getTranslationY()))
+    markerImage.animate().translationY(Math.abs(mapView.getY() / 2))
```

Comment:
Actually, as the `markerImage` has view constraints to its parent, just `markerImage.animate().translationY(0)` should be enough here.

Reply:
Yea, that works. Fixed

![ezgif com-optimize](https://user-images.githubusercontent.com/4394910/42889899-532fbb38-8a60-11e8-9fcf-b33e50dc6188.gif)